application/values.yaml

```diff
@@ -429,8 +429,8 @@ rbac:
 
   # Service Account to use by pods
   serviceAccount:
-    enabled: false
-    name: ""
+    enabled: true
+    name: "application"
```

Comment:
@faizanahmad055 hardcoding to **application**? shouldn't it be a template? what if we deploy five applications in same namespace then they will use the same service account

Reply:
https://github.com/stakater/application/blob/master/application/templates/serviceaccount.yaml#L24

Reply:
in this case we should just remove name from the values.yaml and then it will automatically get the templatized name

Reply:
this should be taken from **helpers.tpl**